src/components/repository-profile.component.js

```diff
@@ -79,10 +79,33 @@ const styles = StyleSheet.create({
   icon: {
     paddingBottom: 20,
   },
+  languageInfo: {
+    flexDirection: 'row',
+    marginTop: 35,
+  },
+  languageInfoTitle: {
+    color: colors.white,
+    paddingLeft: 3,
+    fontSize: normalize(10),
+    ...fonts.fontPrimary,
+  },
 });
 
 export const RepositoryProfile = ({ repository, starred, navigation }: Props) =>
   <View style={styles.container}>
+    {repository.language !== null &&
```

Comment:
I haven't tried with an empty repo (_without a language_). But I was inspired by https://github.com/gitpoint/git-point/blob/master/src/components/repository-list-item.component.js#L86 (Moreover, the condition in this file is only for the icon node).